packages/vitest/src/runtime/map.ts

```diff
@@ -26,9 +25,13 @@ export function isTest(task: Test | Benchmark): task is Test {
   return task.type === 'test'
 }
 
-export function getBenchmarkFactory(key: Suite): BenchFactory {
+export async function getBenchmarkFactory(key: Suite): Promise<BenchFactory> {
   let benchmark = benchmarkMap.get(key)
   if (!benchmark) {
+    if (!globalThis.EventTarget)
+      await import('event-target-polyfill' as any)
```

Comment:
Please consider putting the `any` after the import call!

Reply:
putting the any after the import call can to fix the types error 🤦